src/Service.php

```
@@ -126,8 +126,11 @@ public function registerAdapters(Satellite\Adapter\FactoryInterface ...$adapters
     {
         foreach ($adapters as $adapter) {
             /** @var Configurator\Adapter $attribute */
-            foreach (expectAttributes($adapter, Configurator\Adapter::class) as $attribute) {
-                $this->addAdapter($attribute, $adapter);
+            try {
+                foreach (expectAttributes($adapter, Configurator\Adapter::class) as $attribute) {
+                    $this->addAdapter($attribute, $adapter);
+                }
+            } catch (MissingAttributeException $exception) {
             }
```

Why does this exception is silenced ?